### ISSUE TITLE
Use imperative verbs for numpydoc sections

### ISSUE THREAD

**Issue by McSinyx** [McSinyx/palace#109]
In continuation of GH-94:  PEP-257 suggests that

> [The one-line docstring] prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

I've just realized that [sphinx.ext.napoleon actually supports this](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections), so let's perform the conversion!

**Comment by Huy-Ngo** [comment]
I'm not sure what is going to be addressed by this issue. Aren't we already using imperative verbs for the doc?

**Comment by McSinyx** [comment]
Not for the numpydoc (sub?)sections:
```console
$ grep -nE 'Returns|Warns|Raises' src/palace.pyx 
182:    Raises
206:    Raises
312:    Raises
335:    Raises
354:    Raises
463:    Raises
468:    Warns
688:    Raises
860:        Raises
893:    Raises
977:    Raises
1045:        Raises
1180:    Raises
1359:        Raises
1374:        Raises
1399:        Raises
1431:        Raises
1505:        Raises
1537:        Raises
1567:        Raises
1586:        Raises
1616:        Raises
1829:    Raises
1875:        Raises
1976:    Raises
2078:    Raises
2436:    Raises
2537:    Raises
2610:        Returns
2666:        Returns
2735:        Returns
```

I took a closer look and it's a shame that [there's no imperative alias for *raises* and *warns*](https://github.com/sphinx-doc/sphinx/blob/66e55a02d125e7b65b211a7cf9b48506195e3bf4/sphinx/ext/napoleon/docstring.py#L144-L175), so maybe an PR to Sphinx would be a good idea.

**Comment by Huy-Ngo** [comment]
Ah I see, I was confused about what you mean by the numpydoc section.

**Comment by McSinyx** [comment]
Let's wait for Sphinx 3.2 to be released and solve this, publish 0.2.1 and regenerate documentation altogether.
